src/formats/mmCIF.cpp

```diff
@@ -122,16 +122,15 @@ void mmCIFFormat::init_() {
                     trimmed.substr(1, trimmed.size() - 2).to_string() : "";
         }
 
-        if (in_loop && line_split[0].find("_atom_site") != std::string::npos) {
+        if (in_loop && line_split[0].find("_atom_site.") != std::string::npos) {
```

Comment:
Why have you added the dot? This this just to improve specificity?

Reply:
Exactly. The issue that prompt these change was that `1AKE` contains data in the `_atom_sites.<XXX>` group, which this code would match. For mmCIF files everything related to atoms is inside the `_atom_site.<XXX>` group, hence this change.